variables.tf

```diff
@@ -85,4 +85,9 @@ variable "cluster_parameter_group_name" {
 variable "performance_insights_enabled" {
   default     = false
   description = "Enable performance insights on all database instances in the cluster. Defaults to false"
-}
\ No newline at end of file
+}
+
+variable "enabled_cloudwatch_logs_exports" {
+  default     = null
```

Comment:
Would recommend adding a type if this is a list, and default to empty list instead of null.

Reply:
Fixed